test/Microsoft.AspNet.Mvc.Extensions.Test/Rendering/HtmlHelperTest.cs

```diff
@@ -54,5 +146,135 @@ public class HtmlHelperTest
             var entry = Assert.Single(result);
             Assert.Equal(expectedEntry, entry);
         }
+
+        [Theory]
+        [MemberData(nameof(EncodeDynamicTestData))]
+        public void EncodeDynamic_ReturnsExpectedString(object value, string expectedString)
+        {
+            // Arrange
+            dynamic dynamicValue = value;
+            IHtmlHelper<DefaultTemplatesUtilities.ObjectTemplateModel> helper =
```

Comment:
No that would break the test or at least render it useless. Intent here is to make sure things work when the compiler treats the `HtmlHelper` instance as a specific `interface`.